apps/site/pages/docs/building-sections/overrides/overrides-troubleshooting.mdx

```diff
@@ -0,0 +1,29 @@
+<header>
+# Troubleshooting
+</header>
+
+## I can't display my overrides
+
+### 1. Missing `getOverriddenSection` function
+
+**Symptom:** Overrides fail to display in your store.
+
+**Solution:** Ensure that you have imported the `getOverriddenSection` function from the `@faststore/core` package into your custom component file.
```

Comment:
Maybe we can refer to the setup step from https://faststore-site-git-docs-override-version-2-faststore.vercel.app/docs/building-sections/overrides/override-component-props#step-1---setting-up-the-component-file